data/contract_docs/logs.md

```diff
@@ -0,0 +1,92 @@
+<!-- markdownlint-configure-file {
+  "MD013": false,
+  "MD033": false
+} -->
+
+# Logs
+
+Status: EARLY Draft/Proposal
+
+## LogLines
+
+Version: 0.0
+
+### Properties and field requirements
+
+- **Time field** - _required_
+  - There must be at least one non nullable time field
+  - If there are multiple time fields present, following will decide the priority
+    - First matching time field with name `timestamp`
+- **Message field** - _required_
+  - There must be at lease one non nullable string field must present
+  - If more than one string fields found, the following will decide the priority
+    - First matching string field with name `body`
```

Comment:
i'm OK with the logic here, but i'd like to be sure we describe these cases unambigously.
so, let's say about the `message-field` (timestamp is the same):
- we take every string-field in the dataframe, based on the count of items in the array:
    - `0`. this is an error
    - `1`. this will be the message field
    - `more than 1`. we find all string-fields with the name `body`, based on the count of items in the array:
        - `0`. error? FIXME: what should we do here?
        - `more than 0`. we take the first in order.

Reply:
regarding the `there are multiple string fields, but no string with the name "body"`, i am totally fine with making this into an `error` case, if you're OK with that.

Reply:
i created two separate PRs with two ways we could improve this ( https://github.com/grafana/grafana-plugin-sdk-go/pull/661 and https://github.com/grafana/grafana-plugin-sdk-go/pull/660 ), we can continue the discussion in those, marking this conversation as resolved.